packages/driver/cypress/integration/e2e/origin/snapshots.spec.ts

```diff
@@ -0,0 +1,89 @@
+// import to bind shouldWithTimeout into global cy commands
```

Comment:
What do you mean by this? isn't this  picked up in the the global via the support file?

Reply:
It is not. We don't import `utils` inside the support directory in the `index` or `defaults`. Not entirely sure why, because this seems like it should be included there...